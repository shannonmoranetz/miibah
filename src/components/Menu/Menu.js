import React from 'react';
import { NavLink } from 'react-router-dom';

const Menu = () => {
  return (
    <div className="Menu">
      <NavLink to='/' className="nav-button home-button">home</NavLink>
      <NavLink to='/wishlist' className="nav-button wishlist-button">wishlist</NavLink>
      <NavLink to='/collected' className="nav-button collection-button">collected</NavLink>
    </div>
  );
}

export default Menu;