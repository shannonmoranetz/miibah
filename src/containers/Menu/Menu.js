import React, { Component } from 'react';
import { NavLink } from 'react-router-dom';

class Menu extends Component {
  render() {
    return (
      <div className="Menu">
        <NavLink to='/' className="nav-button home-button">home</NavLink>
        <NavLink to='/' className="nav-button wishlist-button">wishlist</NavLink>
        <NavLink to='/' className="nav-button collection-button">collected</NavLink>
      </div>
    );
  }
}

export default Menu;
