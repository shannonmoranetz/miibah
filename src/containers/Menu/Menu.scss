.Menu {
  text-align: center;
  margin: 40px auto;
}

.nav-button {
  margin: 30px;
  border: 5px solid white;
  padding: 15px;
  font-family: $heading-font;
  font-size: 25px;
  border-radius: 15px;
  background-color: $salmon;
}