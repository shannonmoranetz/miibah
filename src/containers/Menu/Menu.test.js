import Menu from './Menu';

describe('Menu', () => {});