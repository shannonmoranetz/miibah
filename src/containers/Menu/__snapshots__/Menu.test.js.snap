// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should match the snapshot 1`] = `
<div
  className="Menu"
>
  <NavLink
    activeClassName="active"
    aria-current="page"
    className="nav-button home-button"
    to="/"
  >
    home
  </NavLink>
  <NavLink
    activeClassName="active"
    aria-current="page"
    className="nav-button wishlist-button"
    to="/wishlist"
  >
    wishlist
  </NavLink>
  <NavLink
    activeClassName="active"
    aria-current="page"
    className="nav-button collection-button"
    to="/collected"
  >
    collected
  </NavLink>
</div>
`;
